R/tm_variable_browser.R

```diff
@@ -17,6 +18,7 @@
 #' @param datasets_selected (`character`) A vector of datasets which should be
 #'   shown and in what order. Names in the vector have to correspond with datasets names.
 #'   If vector of length zero (default) then all datasets are shown.
+#'   Note: Only datasets of the `data.frame` class are compatible; using other types will cause an error.
```

Comment:
```suggestion
#'   Note: Only datasets of the `data.frame` class are compatible; other data tupés will be skipped.
```

Reply:
Other datatypes for `dataset_selected` will fail assertion. Do we also want to skip other data types for the `dataset_selected `parameter?